src/test_queue.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+"""File tests our queue data structure."""
+from __future__ import unicode_literals
+from queue import Queue
+
+TEST_QUEUE_DATA = [
+    [1, 2, 3, 4, 5],
+    ("cup", "coffee", "pastry")
+]
```

Comment:
This is an insufficient diversity of test cases. Here you're checking only checking a single input of length 2. What about inputs of size 0 or size 1? What about size 1000? How large of a range can your Queue handle?